const orgUserLogin = async (req, res, next) => {
    try{
        console.log("Trying to get orgUserLogin");
    }
    catch(err){
        console.log("Error Getting Org Login: ", err.message);
    }
}